app/src/main/java/com/invisibleteam/goinvisible/util/binding/ImageViewBindingAdapter.java

```diff
@@ -10,18 +10,22 @@
 
 import java.io.File;
 
+import javax.annotation.Nullable;
+
 public class ImageViewBindingAdapter {
-    @BindingAdapter({"bind:imageUrl"})
-    public static void loadImage(ImageView view, ObservableString imageUrl) {
-        Uri uri = Uri.fromFile(new File(imageUrl.get()));
-        Log.d("ImageViewBindingAdapter", uri.toString());
+    @BindingAdapter({"imageUrl"})
+    public static void loadImage(ImageView view, @Nullable ObservableString imageUrl) {
+        if (imageUrl != null) {
```

Comment:
i prefer to check if imageUrl is null and return; instead of encapsulating of the code into if body